pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -346,14 +360,16 @@ type reconciledContainer struct {
 	containerID   string
 }
 
-func (m *manager) removeStaleState() {
+func (m *manager) removeStaleState(ctx context.Context) {
 	// Only once all sources are ready do we attempt to remove any stale state.
 	// This ensures that the call to `m.activePods()` below will succeed with
 	// the actual active pods list.
 	if !m.sourcesReady.AllReady() {
 		return
 	}
 
+	lh_ := klog.FromContext(ctx)
```

Comment:
Why the underscore in `lh_`? Is this a convention? Where is this documented?

Reply:
In this function I'm repeatedly replacing the logger instance using shadowing (lacking any better alternative) in the loop body. I'm keeping a reference of the original passed logger to correctly recreate at each loop iteration, otherwise we unnecessarily grow the context each time. This way each iteration creates a new context with a fixed size payload.

The trailing underscore is the cheapest way to track the original variable: IMO `lhOrig` or `origLogHandle` are unnecessarily long and not really easier to understand.

I'm using the underscore name for the instance which is used less often: we use the function parameter only here.